### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PotuRevanth**
Improved readme and added installation methods.

### CODE REVIEW

**Review by IvanIsCoding — COMMENTED** [review]
Thanks for making the first community PR to the project. Overall looks good but you will need to tweak the installation instructions slightly

**Comment by PotuRevanth** [comment]
> Thanks for making the first community PR to the project. Overall looks good but you will need to tweak the installation instructions slightly

done the necessary changes merge these changes to your project.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ streamlit run src/Main.py
 
 Notice that you will need to install the dependencies in `requirements.txt` for your code to work, and install the packages in `packages.txt` for the LaTeX rendering to work (or equivalent if not using Ubuntu).
 
+#INSTALLATION
```
Comment (IvanIsCoding) [inline README.md:30]:
```suggestion
#I 🪄 Installation
```

File: README.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ streamlit run src/Main.py
...
+
+$ sudo apt-get install package-name
```
Comment (IvanIsCoding) [inline README.md:32]:
```suggestion
$ sudo apt-get install < packages.txt
```

**Review by IvanIsCoding — APPROVED** [review]

**Pull request merged**